src/dlr_tvm.cc

```diff
@@ -228,8 +227,7 @@ bool TVMModel::HasMetadata() const { return !this->metadata.is_null(); }
 
 const char* TVMModel::GetOutputName(const int index) const {
   if (!this->HasMetadata()) {
-    LOG(INFO) << "No metadata file was found!";
-    return nullptr;
+    throw dmlc::Error("No metadata file was found!");
```

Comment:
What happens during `_fetch_output_names()` when there is no metadata file?

Reply:
We return -1 in the C APIs which will throw an error in python bindings.

Reply:
https://github.com/neo-ai/neo-ai-dlr/blob/ef33b6687bae6e084fa077b51bc45efaa054318b/python/dlr/dlr_model.py#L212